MLAB-NS_PRIVACY_POLICY.md

```diff
@@ -0,0 +1,106 @@
+# mlab-ns Privacy Policy
+
+## User-facing Disclaimer
+
+**Context:** A link to this policy should be provided from wherever users are able to run a test which uses mlab-ns.
+
+**Text:** This tool uses mlab-ns, M-Lab's name server. mlab-ns allows users' individual tests to be directed to the appropriate M-Lab server. Like any standard website, mlab-ns receives data over HTTP and collects minimal data for each request. This data is used to debug and improve mlab-ns. The data includes the client's IP address and it does not include location or other personally identifying information. The data is not released publicly. You can find the complete list of all the data collected by mlab-ns at http://mlab-ns.appspot.com/privacy.
+
+## Data received and stored by mlab-ns
+
+mlab-ns receives a number of data fields from each HTTP request. However, it does not store all of the fields received in the request. 
+
+Like any other Google App Engine (GAE) instance, mlab-ns writes to two logs: an application log and a request log.
+
+* The **application log** contains messages written by GAE itself about the application. Hosted applications may add additional content to this log if they wish. 
+* The **request log** is automatically written by GAE when an HTTP request is received before handing it off to the application. Only GAE writes to the request log. The log is similar to the access log written by any HTTP server.
+
+The sections below described what GAE writes to these two logs, focusing primarily on content that has implications on user privacy, along with additional operational information that mlab-ns additionally writes to the application log.
+
+1. mlab-ns receives users' requests over HTTP.
+
+  * Every HTTP request contains the fields listed here: [https://developers.google.com/appengine/docs/python/tools/webapp/requestclass](https://developers.google.com/appengine/docs/python/tools/webapp/requestclass)
+  * Google App Engine also automatically adds to each incoming request the following fields, that describe the **end-user location**.
+    * X-AppEngine-City
+    * X-AppEngine-Region
+    * X-AppEngine-Country
+    * X-AppEngine-CityLatLong
+  * mlab-ns also accepts additional user-defined arguments as part of the URL's query part, as described in the [mlab-ns design doc](DESIGN_DOC.md), all of which are optional, and listed below. User-defined arguments could be defined **by the end-user or by the M-Lab tool developer**.
```

Comment:
I believe you can also pass an `ip` parameter?

mlab-ns.appspot.com/ndt?ip=5.5.5.5

Reply:
Thanks @nkinkade ! I added an another commit to note that `ip` is also a query parameter.